robot_controllers_interface/src/controller_manager.cpp

```diff
@@ -254,7 +254,7 @@ void ControllerManager::execute(const robot_controllers_msgs::QueryControllerSta
     robot_controllers_msgs::ControllerState state = goal->updates[i];
 
     // Make sure controller exists
-    bool exists = false;
+    bool in_default_controllers = false;
```

Comment:
we can probably leave this as "exists" or "in_controller_list" -- technically, once you load your new controller, the "controllers_" list actually consists not just of the default controllers, but also of everything you have loaded since startup